packages/modular-scripts/src/build/index.ts

```diff
@@ -138,10 +137,20 @@ async function buildApp(target: string) {
   )) as CoreProperties;
   targetPackageJson.dependencies = packageDependencies;
   targetPackageJson.bundledDependencies = Object.keys(packageDependencies);
-  // Copy package.json over
+  // Copy selected fields of package.json over
   await fs.writeJSON(
     path.join(paths.appBuild, 'package.json'),
-    targetPackageJson,
+    {
+      name: targetPackageJson.name,
+      version: targetPackageJson.version,
+      main: targetPackageJson.main,
```

Comment:
Where should this point once the app has been built? Same for `module` and `typings`?

Reply:
CRA omits this, which makes sense, since apps don't need importing / requiring